NitroxPatcher/Patches/Dynamic/ExosuitClawArm_OnPickup_Patch.cs

```diff
@@ -22,7 +22,8 @@ public static bool Prefix(ExosuitClawArm __instance)
                 if (pickupable != null && pickupable.isPickupable && componentInParent.storageContainer.container.HasRoomFor(pickupable))
                 {
                     NitroxServiceLocator.LocateService<Item>().PickedUp(pickupable.gameObject, pickupable.GetTechType());
-                } else if(component != null)
+                }
+                else if (component != null)
```

Comment:
```suggestion
                else if (component)
```

Reply:
If this is nullable, you'll need to `if (component == true)`.